templates/migrate.yaml.j2

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: "{{ k8s_namespace }}"
+  name: migrate
+spec:
+  template:
+    spec:
+      containers:
+      - name: "{{ k8s_container_name }}"
+        image: "{{ k8s_container_image }}:{{ k8s_container_image_tag }}"
+        imagePullPolicy: "{{ k8s_container_image_pull_policy }}"
+        command: ["python", "manage.py", "migrate", "--noinput", "-v", "2"]
```

Comment:
This strikes me as something we might want to override in a variable at some point.